app/frontend/next.config.js

```diff
@@ -1,4 +1,11 @@
 /** @type {import('next').NextConfig} */
-const nextConfig = {}
+
```

Comment:
I believe .next folder in /app/frontend should be ignored, otherwise I get a lot of files to commit after running `next dev`.
![image](https://github.com/NethermindEth/l2-finality/assets/4650832/4dbaede8-ecd6-4b3c-a106-ded7e5353e11)